mslib/mscolab/server.py

```diff
@@ -322,6 +322,28 @@ def delete_user():
     return jsonify({"success": True}), 200
 
 
+@APP.route("/edit_full_name", methods=["POST"])
+@verify_user
+def edit_full_name():
+    user = g.user
+    fullname = request.form.get("fullname")
+    name = User.query.filter_by(id=int(user.id)).first()
+    name.fullname = str(fullname)
+    db.session.commit()
+    return jsonify({"success": True}), 200
+
+
+# @APP.route("/edit_nick_name", methods=["POST"])
```

Comment:
maybe we should have only one /edit_profile for the edits of userdata. 

It must be blockable that users can edit once set email Adress and nickname.
That may be can done by some parameter in the mscolab configuration, e.g.
something user_form_disable = []
The server has then to drop changes and the UI may disable the fields.